tools/overlayhelpers/scheduledis.py

```diff
@@ -0,0 +1,241 @@
+    ('SCHEDULE_CMD_BRANCH_L',            0x03, '>h',     (0, )),
+]
+
+scene_names = [
+    'SCENE_20SICHITAI2',
+    'SCENE_UNSET_1',
+    'SCENE_UNSET_2',
+    'SCENE_UNSET_3',
+    'SCENE_UNSET_4',
+    'SCENE_UNSET_5',
+    'SCENE_UNSET_6',
+    'SCENE_KAKUSIANA',
+    'SCENE_SPOT00',
+    'SCENE_UNSET_9',
+    'SCENE_WITCH_SHOP',
+    'SCENE_LAST_BS',
+    'SCENE_HAKASHITA',
+    'SCENE_AYASHIISHOP',
+    'SCENE_UNSET_E',
+    'SCENE_UNSET_F',
+    'SCENE_OMOYA',
+    'SCENE_BOWLING',
+    'SCENE_SONCHONOIE',
+    'SCENE_IKANA',
+    'SCENE_KAIZOKU',
+    'SCENE_MILK_BAR',
+    'SCENE_INISIE_N',
+    'SCENE_TAKARAYA',
+    'SCENE_INISIE_R',
+    'SCENE_OKUJOU',
+    'SCENE_OPENINGDAN',
+    'SCENE_MITURIN',
+    'SCENE_13HUBUKINOMITI',
+    'SCENE_CASTLE',
+    'SCENE_DEKUTES',
+    'SCENE_MITURIN_BS',
+    'SCENE_SYATEKI_MIZU',
+    'SCENE_HAKUGIN',
+    'SCENE_ROMANYMAE',
+    'SCENE_PIRATE',
+    'SCENE_SYATEKI_MORI',
+    'SCENE_SINKAI',
+    'SCENE_YOUSEI_IZUMI',
+    'SCENE_KINSTA1',
+    'SCENE_KINDAN2',
+    'SCENE_TENMON_DAI',
+    'SCENE_LAST_DEKU',
+    'SCENE_22DEKUCITY',
+    'SCENE_KAJIYA',
+    'SCENE_00KEIKOKU',
+    'SCENE_POSTHOUSE',
+    'SCENE_LABO',
+    'SCENE_DANPEI2TEST',
+    'SCENE_UNSET_31',
+    'SCENE_16GORON_HOUSE',
+    'SCENE_33ZORACITY',
+    'SCENE_8ITEMSHOP',
+    'SCENE_F01',
+    'SCENE_INISIE_BS',
+    'SCENE_30GYOSON',
+    'SCENE_31MISAKI',
+    'SCENE_TAKARAKUJI',
+    'SCENE_UNSET_3A',
+    'SCENE_TORIDE',
+    'SCENE_FISHERMAN',
+    'SCENE_GORONSHOP',
+    'SCENE_DEKU_KING',
+    'SCENE_LAST_GORON',
+    'SCENE_24KEMONOMITI',
+    'SCENE_F01_B',
+    'SCENE_F01C',
+    'SCENE_BOTI',
+    'SCENE_HAKUGIN_BS',
+    'SCENE_20SICHITAI',
+    'SCENE_21MITURINMAE',
+    'SCENE_LAST_ZORA',
+    'SCENE_11GORONNOSATO2',
+    'SCENE_SEA',
+    'SCENE_35TAKI',
+    'SCENE_REDEAD',
+    'SCENE_BANDROOM',
+    'SCENE_11GORONNOSATO',
+    'SCENE_GORON_HAKA',
+    'SCENE_SECOM',
+    'SCENE_10YUKIYAMANOMURA',
+    'SCENE_TOUGITES',
+    'SCENE_DANPEI',
+    'SCENE_IKANAMAE',
+    'SCENE_DOUJOU',
+    'SCENE_MUSICHOUSE',
+    'SCENE_IKNINSIDE',
+    'SCENE_MAP_SHOP',
+    'SCENE_F40',
+    'SCENE_F41',
+    'SCENE_10YUKIYAMANOMURA2',
+    'SCENE_14YUKIDAMANOMITI',
+    'SCENE_12HAKUGINMAE',
+    'SCENE_17SETUGEN',
+    'SCENE_17SETUGEN2',
+    'SCENE_SEA_BS',
+    'SCENE_RANDOM',
+    'SCENE_YADOYA',
+    'SCENE_KONPEKI_ENT',
+    'SCENE_INSIDETOWER',
+    'SCENE_26SARUNOMORI',
+    'SCENE_LOST_WOODS',
+    'SCENE_LAST_LINK',
+    'SCENE_SOUGEN',
+    'SCENE_BOMYA',
+    'SCENE_KYOJINNOMA',
+    'SCENE_KOEPONARACE',
+    'SCENE_GORONRACE',
+    'SCENE_TOWN',
+    'SCENE_ICHIBA',
+    'SCENE_BACKTOWN',
+    'SCENE_CLOCKTOWER',
+    'SCENE_ALLEY'
+]
+
+def read_bytes(data_file, offset, len):
+    data_file.seek(offset)
+    return bytearray(data_file.read(len))
+
+def disassemble_unk_script(data_file, offset):
+    off = 0
+    cmd = None
+    branch_targets = []
+
+    out = "static u8 sEventScript[] = {\n"
```

Comment:
```suggestion
    out = "static u8 sScheduleScript[] = {\n"
```
I think it would be better to have `sScheduleScript` so that it matches the name of the file, but that is up to you.

Reply:
+1 for this, especially since there is a separate `eventMgr` file that deals with actor cutscenes, which is separate to this scheduling system